doc/news/changes/major/20230422StefanoZampini

```diff
@@ -0,0 +1,4 @@
+New: Add wrappers to the non-linear solver SNES and
+the ode integrator TS from the PETSc library.
+<br>
```

Comment:
Can you make sure they show up as references here? I.e., 
```suggestion
New: Add wrappers to the non-linear solver SNES (PETScWrappers::NonLinearSolver ) and
the ode integrator TS (PETScWrappers::TimeStepper) from the PETSc library.
<br>
```
This will make them show up as fully functional links to the doxygen documentation page.